js/components/authoring.js

```diff
@@ -180,9 +194,14 @@ export default class Authoring extends PureComponent {
           <div className='divider' />
           { this.renderStep(3) }
           { this.renderInfo(3, 'Assign forces to plates') }
-          <div className='divider' />
-          { this.renderStep(4) }
-          { this.renderInfo(4, 'Order plates by density') }
+          {
+            this.densityStepEnabled &&
```

Comment:
Note that there's a config object available:
https://github.com/concord-consortium/plate-tectonics-3d/blob/master/js/config.js
It automatically handles URL params to overwrite default values. Obviously, URL param needs to have the same name as config property.

So you could just add `densityStepEnabled` to this config object and just import `config` here and type `config.densityStepEnabled &&`. No need to parse URL param by hand.

Reply:
Much cleaner this way, thanks Piotr!